### PR TITLE
Update README with more details for beta 1

### PR DESCRIPTION
**Pull request by rishson**
[skip ci]

Refs: #19

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,16 +20,25 @@ A suite of pre-built Dojo 2 widgets, ready to use in your application. These wid
...
 
 ## Features
 
 Dojo 2 widgets are supported in all evergreen browsers (IE11+, Chrome, Firefox, Safari) as well as popular mobile browsers (Mobile Safari, Chrome on Android).
 
+Dojo 2 widgets are fully accessible, out of the box.
+
+Dojo 2 widgets are fully themeable. In a future release we will ship with several official Dojo 2 themes.
+
+Dojo 2 widgets
```
Comment (dylans) [inline README.md]: This looks like a stray thought?

Comment (rishson) [inline README.md]: yup - removed

**Comment by agubler** [comment]
Are you adding more to this around the specific widgets?

**Comment by rishson** [comment]
Was not going to add individual widget commentary until beta 2

**Comment by agubler** [comment]
I think we could probably start on it now for the widgets we have so far. Will help spread the effort too :)

**Comment by dylans** [comment]
> I think we could probably start on it now for the widgets we have so far. Will help spread the effort too :)

I'd say we land this PR and then do that... that way this PR is at least ready to go for beta1.

**Comment by agubler** [comment]
> I'd say we land this PR and then do that... that way this PR is at least ready to go for beta1.

If this package isn't being released as part of beta1 then im not sure it matters. If it is being released as beta1 then it probably needs these README changes before that.

Either way not opposed to landing this then adding the proper readme

**Pull request merged**